astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -191,14 +191,14 @@ def delete_redshift_cluster_snapshot_callable() -> None:
     )
     # [END howto_operator_redshift_resume_cluster_async]
 
-    # [START howto_operator_redshift_cluster_sensor_async]
+    # [START howto_senor_redshift_cluster_async]
```

Comment:
sensor spelling, change howto_senor_redshift_cluster_async to howto_sensor_redshift_cluster_async